teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, name, parent_permission, specialization)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As requested by the previous reviewer please make sure the `parent_permission` parameter has a default value as `true`.